datafusion/core/tests/sqllogictests/test_files/timestamps.slt

```diff
@@ -317,11 +317,11 @@ select
 ----
 08:09:10.123456789 13:14:15.123456 13:14:15.123 13:14:15
 
-query error Cannot cast string 'not a time' to value of Time64\(Nanosecond\) type\. This was likely caused by a bug in DataFusion's code and we would welcome that you file an bug report in our issue tracker
+query error Cannot cast string 'not a time' to value of Time64\(Nanosecond\) type
 SELECT TIME 'not a time' as time;
 
 # invalid time
-query error Cannot cast string '24:01:02' to value of Time64\(Nanosecond\) type\. This was likely caused by a bug in DataFusion's code and we would welcome that you file an bug report in our issue tracker
+query error Cannot cast string '24:01:02' to value of Time64\(Nanosecond\) type
```

Comment:
I dont see the new text in tests 🤔

Reply:
The text following `query error` is a regular expression, so the test will pass without all the text. I could add the first part of the message, but it wasn't there to start with.

Reply:
nah I was wrong, it covered, and thats why tests are failing
```
---- optimizer::tests::no_skip_failing_rule stdout ----
thread 'optimizer::tests::no_skip_failing_rule' panicked at 'assertion failed: `(left == right)`
  left: `"bad rule\ncaused by\nInternal error: Optimizer rule 'bad rule' failed due to unexpected error: Error during planning: rule failed. This was likely caused by a bug in DataFusion's code and we would welcome that you file an bug report in our issue tracker"`,
 right: `"Optimizer rule 'bad rule' failed\ncaused by\nError during planning: rule failed"`', datafusion\optimizer\src\optimizer.rs:465:9
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

Reply:
Yep, I missed a unit test in the very file I changed. It's fixed now.